cfme/tests/openstack/test_relationships.py

```diff
@@ -61,10 +39,10 @@ def test_templates(provider):
     result = provider.summary.relationships.templates.value
     """
     todo get the list of images/templates from external resource and compare
-    it with result - currently  5
+    it with result - currently  bigger than 0
     """
 
-    assert result == 5
+    assert result > 0
```

Comment:
Sure, I need to get this attribute from the mgmt system. 
Can you point me on which method to use in order to be able to compare the 'real' number of images in RHOS?